lib/utils/compose.js

```diff
@@ -85,23 +85,6 @@ export function createProject(composePath, composeStr, projectName = null) {
 	};
 }
 
-/**
```

Comment:
The deleted code in `compose.js` was converted to Typescript and moved to the pre-existing `compose_ts.ts` file.